src/tlo/methods/alri.py

```diff
@@ -55,7 +56,7 @@
 # ---------------------------------------------------------------------------------------------------------
 
 
-class Alri(Module):
+class Alri(GenericFirstApptModule):
```

Comment:
Do you prefer subclassing over a mixin?

Reply:
Subclassing seemed more natural here to me - no need for multiple inheritance and I think we would only use the mix-in with `Module` as base class anyway so wasn't clear there would be any advantage to a mix-in over subclassing.

Reply:
I thought it'd be easier to pull in other interfaces or methods for \`Module\` (e.g. for disease vs non-disease modules) if needed.

Reply:
Thought about this some more, and I do think we should use mixins here rather than subclass. Flexibility of mixins offer more opportunity to pull out and refactor some common functionality across `Module` classes.

Reply:
I've now refactored to make a mix-in